packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -355,15 +355,15 @@ export function useSelectableCollection(options: SelectableCollectionOptions): S
   }, []);
 
   let handlers = {
-    // We use a capturing listener to ensure that the keyboard events for the collection
-    // override those of the children. For example, ArrowDown in a table should always go
-    // to the cell below, and not open a menu.
     onKeyDown,
     onFocus,
     onBlur,
     onMouseDown(e) {
       // Prevent focus going to the collection when clicking on the scrollbar.
-      e.preventDefault();
+      // e.target.parentElement check is for Table since the scrollbar click occurs on the Table body
+      if (e.target.parentElement === ref.current || e.target === ref.current) {
+        e.preventDefault();
+      }
```

Comment:
Bit iffy about this since it prevents text selection on a the collection's immediate child element (e.g. imagine that the Table body was just a div with text)

Reply:
We should move the table specific case into table so it doesn't apply to other components.